src/test/java/core/basesyntax/tests/TestStorage.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.tests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+
+public class TestStorage {
+    private static Map<Fruit, Integer> data = new HashMap<>();
+    private static Map<Fruit, Integer> storage = Storage.getFruits();
+
+    @BeforeEach
+    public void setValues() {
+        data.put(new Fruit("banana"), 100);
+        data.put(new Fruit("apple"), 15);
+        storage.put(new Fruit("banana"), 100);
+        storage.put(new Fruit("apple"), 15);
+    }
+
+    @Before
+    public void cleanData() {
+        data.clear();
+        storage.clear();
+    }
```

Comment:
@Before and @BeforeEach are both run before each test - they are just from different Junit libraries - let's use only one of them